packages/core/src/components/Dropdown/components/MultiValueContainer/MultiValueContainer.module.scss

```diff
@@ -11,10 +11,18 @@
   --chips-margin: 2px;
 }
 
-.multiselect-chip-single-line {
+.multiselect-chip-single-line-medium {
   --chips-margin: 5px 2px;
 }
 
+.multiselect-chip-single-line-large {
+  --chips-margin: 9px 2px !important;
```

Comment:
Is the important necessary? Let's try to avoid it since it's catch all and can cause specificity issues

Reply:
No, I forgot to remove it, added it just while I was testing, thanks!